src/pkg/variables/templates.go

```diff
@@ -130,5 +129,11 @@ func (vc *VariableConfig) ReplaceTextTemplate(path string) error {
 		}
 	}
 
+	// We're done scanning, close our file
+	err = textFile.Close()
```

Comment:
We should do named parameter defer return to ensure we always get to the close block

Reply:
Yeah, that's fair -- these don't have any returns in the way so they should be handled inline, but there's no harm in guarding against panics.